builder/js/ncf.js

```diff
@@ -591,22 +589,20 @@ $scope.groupMethodsByCategory = function () {
   $scope.checkFilterCategory = function(methods){
```

Comment:
That method should not be applied like this, but using  a filter on the needed ng-repeat https://docs.angularjs.org/api/ng/filter/filter